test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Rendering/HtmlHelperValidationSummaryTest.cs

```diff
@@ -109,15 +109,15 @@ public class HtmlHelperValidationSummaryTest
                     "<li>HtmlEncode[[This is an error for Property3.]]</li>" + Environment.NewLine +
                     "</ul></div>";
                 var divWithAllErrors = "<div class=\"HtmlEncode[[validation-summary-errors]]\" data-valmsg-summary=\"HtmlEncode[[true]]\"><ul>" +
-                    "<li>HtmlEncode[[This is an error for Property3.Property2.]]</li>" + Environment.NewLine +
-                    "<li>HtmlEncode[[This is an error for Property3.OrderedProperty3.]]</li>" + Environment.NewLine +
-                    "<li>HtmlEncode[[This is an error for Property3.OrderedProperty2.]]</li>" + Environment.NewLine +
-                    "<li>HtmlEncode[[This is an error for Property3.]]</li>" + Environment.NewLine +
+                    "<li>HtmlEncode[[This is an error for the model root.]]</li>" + Environment.NewLine +
+                    "<li>HtmlEncode[[This is another error for the model root.]]</li>" + Environment.NewLine +
                     "<li>HtmlEncode[[This is an error for Property2.]]</li>" + Environment.NewLine +
                     "<li>HtmlEncode[[This is another error for Property2.]]</li>" + Environment.NewLine +
                     "<li>HtmlEncode[[The value '' is not valid for Property2.]]</li>" + Environment.NewLine +
-                    "<li>HtmlEncode[[This is an error for the model root.]]</li>" + Environment.NewLine +
-                    "<li>HtmlEncode[[This is another error for the model root.]]</li>" + Environment.NewLine +
+                    "<li>HtmlEncode[[This is an error for Property3.]]</li>" + Environment.NewLine +
+                    "<li>HtmlEncode[[This is an error for Property3.OrderedProperty2.]]</li>" + Environment.NewLine +
+                    "<li>HtmlEncode[[This is an error for Property3.OrderedProperty3.]]</li>" + Environment.NewLine +
+                    "<li>HtmlEncode[[This is an error for Property3.Property2.]]</li>" + Environment.NewLine +
```

Comment:
We're not ordering by name. There is no implicit ordering between root, `Property2`, and `Property3` (they use the default order). We ignore the order attribute on sub-properties so there isn't specific ordering between `Property3.OrderedProperty2`, `Property3.OrderedProperty3` and `Property3.Property2`.